tensorflow/core/kernels/reverse_op.cc

```diff
@@ -149,9 +145,13 @@ TF_CALL_double(DECLARE_GPU_SPEC);
                           ReverseOp<GPUDevice, T>)
 TF_CALL_uint8(REGISTER_GPU_KERNEL);
 TF_CALL_int8(REGISTER_GPU_KERNEL);
+//TF_CALL_int32(REGISTER_GPU_KERNEL);
+//TF_CALL_bool(REGISTER_GPU_KERNEL);
```

Comment:
I've tried enabling them, but had a problem with `int32`.
The tests were crashing without an error message for me :/
So I wasn't quite sure what to put in the TODO comment.
Should I add a comment saying `// TODO Find out why int32 doesn't work` ?